tensorflow_serving/tools/docker/tests/BUILD

```diff
@@ -46,6 +46,38 @@ sh_test(
     deps = [":docker_test_lib"],
 )
 
+sh_test(
+    name = "unittest_dockerfile_mkl",
+    size = "medium",
+    srcs = ["dockerfile_mkl_test.sh"],
+    args = ["tensorflow/serving:latest-mkl"],
+    data = [
+        "//tensorflow_serving/servables/tensorflow/testdata:saved_model_half_plus_two_cpu",
+    ],
+    tags = [
+        "exclusive",
+        "local",
+        "manual",
+    ],
+    deps = [":docker_test_lib"],
+)
+
+sh_test(
+    name = "unittest_dockerfile_devel_mkl",
+    size = "medium",
```

Comment:
Same here - this size could be small hopefulluy

Reply:
yep, will update.